decidim-comments/app/frontend/comments/comments.component.tsx

```diff
@@ -67,6 +69,54 @@ export class Comments extends React.Component<CommentsProps> {
     );
   }
 
+  /**
+   * Renders warning message when viewing a single comment.
+   * @private
+   * @returns {Void|DOMElement} - A warning message or nothing.
+   */
+  private _renderSingleCommentWarning() {
+    const { singleCommentId, reorderComments, orderBy } = this.props;
+
+    if (singleCommentId && singleCommentId !== "") {
+      const newUrl = `${window.location.pathname}${window.location.search.replace(`commentId=${singleCommentId}`, "")}`;
```

Comment:
Can we jump directly to the comments section this way?
```suggestion
      const newUrl = `${window.location.pathname}${window.location.search.replace(`commentId=${singleCommentId}`, "#comments")}`;
```

Reply:
I tried, but I couldn't get it working because the `#comments` element is added by React once the comments are loaded, and the `#comments` part in the URL is executed once the page is loaded (but before React is done), so it can't find it and it doesn't jump to the section... 😓